tools/Google.Cloud.Tools.VersionCompat.Tests/TopLevelTest.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Xunit;
+
+namespace TopLevel
+{
+    namespace ClassUnchanged.A { public class C { } }
```

Comment:
It's a pity we can't add attributes to express the expected result inline. I'll think about whether there's any nice way we could do this - unless there's some more fundamental issue that I'm missing that would stop us from doing it.

Reply:
Yes, agreed this could be nice, although I can't see an obvious way to do that right now!